src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+import duke.command.EmptyCommand;
+import duke.command.ICommand;
+
+/**
+ * Represents the main class of the Duke program.
+ */
+public class Duke {
+    private final Storage storage;
+    private final TaskList taskList;
```

Comment:
I am confused why this taskList should be final because it will be changed and reload, right? 🤔

Reply:
Understand, I think you're right here. It's also ambiguous to label it as final where it's not a constant.